### PR TITLE
MAINT update readme with paper

### LINKED ISSUE #610: ENH website
update https://github.com/kymatio/kymatio#how-to-cite with http://jmlr.org/papers/v21/19-047.html

### PR DESCRIPTION
**Pull request by MuawizChaudhary**
This Pull Request updates our README with the paper submitted to JMLR. 

This closes #610

### CODE REVIEW

**Comment by MuawizChaudhary** [comment]
How do we wanna address issue #596 with this PR?

File: README.md [hunk 1]
```diff
@@ -46,9 +46,24 @@ We refer to our [official benchmarks](https://www.kymat.io/userguide.html#benchm
 
 ## How to cite
 
-If you use this package, please cite the following paper:
-
-Andreux M., Angles T., Exarchakis G., Leonarduzzi R., Rochette G., Thiry L., Zarka J., Mallat S., Andén J., Belilovsky E., Bruna J., Lostanlen V., Hirn M. J., Oyallon E., Zhang S., Cella C., Eickenberg M. (2019). Kymatio: Scattering Transforms in Python. arXiv preprint arXiv:1812.11214. [(paper)](https://arxiv.org/abs/1812.11214)
+If you use this package, please cite our paper <a href="">Kymatio: Scattering Transforms in Python</a>:
```
Comment (janden) [inline README.md:49]: Can we keep the traditional citation and include a link to the BibTeX entry? Looks quite ugly (and not all of us are fluent in BibTeX).

File: README.md [hunk 2]
```diff
@@ -46,9 +46,24 @@ We refer to our [official benchmarks](https://www.kymat.io/userguide.html#benchm
...
+
+```
+@article{JMLR:v21:19-047,
+  author  = {Mathieu Andreux and TomÃ¡s Angles and Georgios Exarchakis and
```
Comment (janden) [inline README.md:53]: Please fix encoding here.

**Comment by eickenberg** [comment]
agree with joakim's review

**Comment by MuawizChaudhary** [comment]
Look good?

File: README.md [hunk 3]
```diff
@@ -46,24 +46,10 @@ We refer to our [official benchmarks](https://www.kymat.io/userguide.html#benchm
...
-  title   = {Kymatio: Scattering Transforms in Python},
-  journal = {Journal of Machine Learning Research},
-  year    = {2020},
-  volume  = {21},
-  number  = {60},
-  pages   = {1-6},
-  url     = {http://jmlr.org/papers/v21/19-047.html}
-}
-```
+If you use this package, please cite the following paper:	If you use this package, please cite our paper <a href="">Kymatio: Scattering Transforms in Python</a>:
```
Comment (janden) [inline README.md:49]: Remove repetition.

**Pull request merged**